hack/e2e.go

```diff
@@ -563,6 +568,9 @@ func KubemarkTest() error {
 	if err != nil {
 		return err
 	}
+	// If we tried to bring the Kubemark cluster up, make a courtesy
+	// attempt to bring it down so we're not leaving resources around.
+	defer finishRunning("Stop kubemark", exec.Command("./test/kubemark/stop-kubemark.sh"))
```

Comment:
Is it important to stop kubemark separately from turning down the cluster? Otherwise isn't this handled by Down()?